### PR TITLE
fix: set auto commit back to true after add/remove

### PR DESCRIPTION
**Pull request by JCereal**
Fix: https://github.com/jcasbin/jdbc-adapter/issues/51

### CODE REVIEW

File: src/main/java/org/casbin/adapter/JDBCBaseAdapter.java [hunk 1]
```diff
@@ -354,8 +354,15 @@ public void addPolicies(String sec, String ptype, List<List<String>> rules) {
                 if(count!=0){
                     ps.executeBatch();
                 }
+                conn.commit();
+            } catch (SQLException e) {
+                conn.rollback();
+
+                e.printStackTrace();
+                throw e;
```
Comment (tangyang9464) [inline src/main/java/org/casbin/adapter/JDBCBaseAdapter.java:362]: It's already caught here, no need to ``throw e`` anymore

**Review by tangyang9464 — CHANGES_REQUESTED** [review]

File: src/main/java/org/casbin/adapter/JDBCBaseAdapter.java (hunk 1 above)
Comment (JCereal) [reply src/main/java/org/casbin/adapter/JDBCBaseAdapter.java:362]: My understanding is, if not throwing out the exception here, then jcasbin won't know the add policy operation actually failed in the adapter: https://github.com/casbin/jcasbin/blob/master/src/main/java/org/casbin/jcasbin/main/InternalEnforcer.java#L45-L47

Comment (tangyang9464) [reply src/main/java/org/casbin/adapter/JDBCBaseAdapter.java:362]: you're right.

**Review by tangyang9464 — APPROVED** [review]
lgtm

**Pull request merged**